### DISCUSSION TITLE
Add a option to extend current subscription

### DISCUSSION THREAD

**Discussion by ghost** [anonaddy/anonaddy#358] [Feature Requests]
A lot of services let users to extend their current subscription without the need to upgrade the current plan.

I think it's a must for Anonaddy and preliminarily it doesn't looks too hard to implement.

With that functionality, there's no need to keep credit card records attached to user accounts.

**Comment by willbrowningme** [comment]
This is already possible if you subscribe using either cryptocurrency or by buying voucher codes, you can extend your current subscription at any point.

**Reply by ghost** [reply]
Nice! Good to know!
But that isn't clear at the site...
And the credit card attached to accounts still there...
Thx!

**Reply by willbrowningme** [reply]
If you purchase a voucher code the card is not saved to your account.

It is only saved when paying for a recurring subscription so that the subscription can be automatically renewed.

**Reply by ghost** [reply]
Can't remove card. Cancelled my current subscription and bought an voucher but can't use it. 

![image](https://user-images.githubusercontent.com/4828145/199210913-0ed7074d-7ec8-4845-8829-23247082050b.png)

**Reply by willbrowningme** [reply]
Please [send me an email](https://anonaddy.com/contact/) with details of your username so I can sort this for you.
